### PR TITLE
Add view

### PR DESCRIPTION
**Pull request by vishandi**

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -122,12 +124,23 @@ public void setPerson(Person target, Person editedPerson) {
         return filteredPersons;
     }
 
+    @Override
+    public ObservableList<Person> getViewedPersonList() {
```
Comment (tyanhan) [inline src/main/java/seedu/address/model/ModelManager.java:128]: Likewise, we can rename this to getViewedSchedule?

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -121,6 +125,9 @@ void fillInnerParts() {
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+
+        viewListPanel = new PersonListPanel(logic.getViewedPersonList()); //TODO: CHANGE THIS!!!!!!
```
Comment (tyanhan) [inline src/main/java/seedu/address/ui/MainWindow.java:129]: Remember to remove the comment later on!

**Review by tyanhan — CHANGES_REQUESTED** [review]
Great job, its pretty awesome that you managed to finish it so quickly. It would be nice if you could add a screenshot of the updated UI as a comment.

File: src/main/java/seedu/address/model/ModelManager.java [hunk 3]
```diff
@@ -122,12 +124,23 @@ public void setPerson(Person target, Person editedPerson) {
...
+    }
+
     @Override
     public void updateFilteredPersonList(Predicate<Person> predicate) {
         requireNonNull(predicate);
         filteredPersons.setPredicate(predicate);
     }
 
+    @Override
+    public void updateViewedPersonList(Predicate<Person> predicate) {
```
[the remark this answers is not in the document]
Comment (vishandi) [reply src/main/java/seedu/address/model/ModelManager.java:139]: I think it is to view the whole person card, but with a detailed version of that person's schedule

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 2 above)
Comment (vishandi) [reply src/main/java/seedu/address/ui/MainWindow.java:129]: okay! need to do this modification anyway before submitting the project

File: src/main/java/seedu/address/model/ModelManager.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (tyanhan) [reply src/main/java/seedu/address/model/ModelManager.java:139]: So in the future we will remove the list of events from the PersonCard in the main contact list, and only show it on the right panel right?

[the remark this answers is not in the document]
Comment (vishandi) [reply src/main/java/seedu/address/model/ModelManager.java:139]: could be, or we could somehow summarize the list of events from the personcard in the main contact list. i think we could discuss that in the next meeting

**Pull request merged**